pkg/client/images.go

```diff
@@ -61,3 +62,10 @@ func (c *images) Delete(name string) (err error) {
 	err = c.r.Delete().Resource("images").Name(name).Do().Error()
 	return
 }
+
+// UpdateSignatures allows to modify signatures of given image.
+func (c *images) UpdateSignatures(image *imageapi.Image) (result *imageapi.Image, err error) {
```

Comment:
>> NamedCreater. I don't know if someone removed the NamedUpdater equivalent or not.
>
> @miminar that's the one you want. And since its a different type, POST works out very cleanly.

It makes certainly sense from the API's point of view. However, I'm struggling to find an appropriate format for the signature name:

1. Shall it be a composition of `Type` and `Content`?
  - The `Content` may be quite long - I certainly wouldn't like to be named like that. And the redundancy.
2. Shall we make some digest of `<type>:<content>` instead?
  - That's client unfriendly.
3. So shall it be some auto-generated string?

If we solve that, the `Create` part looks easy. But I'm not sure about signature removal. I can't find anything like `NamedDeleter` (only in upstream comment). Does it mean we have to create it upstream?

```go
func (c *images) CreateSignature(imageName string, signature *imageapi.ImageSignature) (result *imageapi.ImageSignature, err error) {}
func (c *images) DeleteSignature(imageName, name string) (err error) {}
```